src/sentry/integrations/client.py

```diff
@@ -193,37 +197,48 @@ def _request(
             sample_rate=1.0,
             tags={"integration": self.integration_name},
         )
-        try:
-            resp = getattr(session, method.lower())(
-                url=full_url,
-                headers=headers,
-                json=data if json else None,
-                data=data if not json else None,
-                params=params,
-                auth=auth,
-                verify=self.verify_ssl,
-                allow_redirects=allow_redirects,
-                timeout=timeout,
-            )
-            resp.raise_for_status()
-        except ConnectionError as e:
-            self.track_response_data("connection_error", e)
-            raise ApiHostError.from_exception(e)
-        except Timeout as e:
-            self.track_response_data("timeout", e)
-            raise ApiTimeoutError.from_exception(e)
-        except HTTPError as e:
-            resp = e.response
-            if resp is None:
-                self.track_response_data("unknown", e)
-                self.logger.exception(
-                    "request.error", extra={"integration": self.integration_name, "url": full_url}
+
+        with sentry_sdk.start_span(
+            op="integrations.http",
+            transaction=u"integrations.http_response.{}".format(self.integration_name),
+        ) as span:
+            try:
+                resp = getattr(session, method.lower())(
+                    url=full_url,
+                    headers=headers,
+                    json=data if json else None,
+                    data=data if not json else None,
+                    params=params,
+                    auth=auth,
+                    verify=self.verify_ssl,
+                    allow_redirects=allow_redirects,
+                    timeout=timeout,
                 )
-                raise ApiError("Internal Error")
-            self.track_response_data(resp.status_code, e)
-            raise ApiError.from_response(resp)
+                resp.raise_for_status()
+            except ConnectionError as e:
+                span.set_status("unavailable")
+                self.track_response_data("connection_error", span, e)
+                raise ApiHostError.from_exception(e)
+            except Timeout as e:
+                span.set_status("deadline_exceeded")
+                self.track_response_data("timeout", span, e)
+                raise ApiTimeoutError.from_exception(e)
+            except HTTPError as e:
+                resp = e.response
+                if resp is None:
+                    span.set_status("unknown_error")
```

Comment:
@MeredithAnya Is there a reason why the span status is different than the status we put into datadog and logging?

Reply:
sorry forgot to push up my latest change